### PR TITLE
Update tutorial instructions to use icon for adding Home children pages

### PR DESCRIPTION
**Pull request by cliff688**
Updates tutorial to reflect 'Add child page' as a primary action with a + icon on the Home page in admin, replacing the previous ... option, following the change in 58d6c137b44c.

### CODE REVIEW

**Comment by lb-** [comment]
Someone on Slack ran into this, I'll try to review soon as it seems to be causing confusion.

File: docs/getting_started/tutorial.md [hunk 1]
```diff
@@ -342,7 +342,7 @@ Now that this is complete, here is how you can create a page from the Wagtail [a
 
 1.  Go to <http://127.0.0.1:8000/admin> and sign in with your admin user details.
 2.  In the Wagtail [admin interface](https://guide.wagtail.org/en-latest/concepts/wagtail-interfaces/#admin-interface), go to Pages, then click Home.
-3.  Add a child page to the Home page by clicking **...** at the top of the screen and selecting the option **Add child page**.
+3.  Add a child page to the Home page by clicking the **`+`** icon at the top of the screen.
```
Comment (lb-) [inline docs/getting_started/tutorial.md:345]:
```suggestion
3.  Add a child page to the Home page by clicking the **`+`** icon (Add child page) at the top of the screen.
```

I think we should add what the button is here, I will update this when doing the merge, plus fix some other whitespace issues on this page.

File: CHANGELOG.txt [hunk 2]
```diff
@@ -36,6 +36,7 @@ Changelog
  * Docs: Document usage of custom validation for admin form pages when using `AbstractEmailForm` or `AbstractForm` pages (John-Scott Atlakson, LB (Ben) Johnston)
  * Docs: Reword `BlogTagIndexPage` example for clarity (Clifford Gama)
  * Docs: Change title of blog index page in tutorial to avoid slug issues (Thibaud Colas)
+ * Docs: Update tutorial to use `+` icon for adding child pages to Home page (Clifford Gama)
```
Comment (lb-) [inline CHANGELOG.txt:39]: Not a big issue but in the future, there's no need to include the chnagelog entry, the core team will do this when merging.

Thanks though.

File: docs/tutorial/create_contact_page.md [hunk 3]
```diff
@@ -121,9 +121,8 @@ To add contact information to your portfolio site, follow these steps:
     b. Go to your admin interface.  
     c. Click `Pages` in your [Sidebar](https://guide.wagtail.org/en-latest/how-to-guides/find-your-way-around/#the-sidebar).  
     d. Click `Home`.  
-    e. Click the `...` icon at the top of the resulting page.  
-    f. Click `add child page`.  
-    g. Click `Form page`.
+    e. Click the `+` icon at the top of the resulting page.
```
Comment (lb-) [inline docs/tutorial/create_contact_page.md:124]:
```suggestion
    e. Click the `+` icon (Add child page) at the top of the resulting page.
```

File: docs/tutorial/create_portfolio_page.md [hunk 4]
```diff
@@ -335,9 +335,8 @@ To add your resume to your portfolio site, follow these steps:
     b. Go to your admin interface.  
     c. Click `Pages` in your [Sidebar](https://guide.wagtail.org/en-latest/how-to-guides/find-your-way-around/#the-sidebar).  
     d. Click `Home`.  
-    e. Click the `...` icon at the top of the resulting page.  
-    f. Click `add child page`.  
-    g. Click `Portfolio Page`.
+    e. Click the `+` icon at the top of the resulting page.
```
Comment (lb-) [inline docs/tutorial/create_portfolio_page.md:338]:
```suggestion
    e. Click the `+` icon (Add child page) at the top of the resulting page.
```

**Review by lb- — APPROVED** [review]
Thank you @cliff688  I have added those minor tweaks and will force push, then once the CI passes I will merge.

**Comment by cliff688** [comment]
Thanks @lb- for the review

**Comment by gasman** [comment]
Thanks @cliff688 and @lb-! Have now cherry-picked this to the stable/6.3.x branch too (in 7470051d22542a8956395ec260a1907a557da233), to be included in a potential future 6.3.2 release.

**Pull request merged**